examples/index.json

```diff
@@ -2433,6 +2433,14 @@
         "verifyOutput": false,
         "disablePlayground": true,
         "isLearnByExample": false
+      },
+      {
+        "name": "ID Scalar Type",
+        "url": "graphql-id-scalar",
+        "verifyBuild": true,
+        "verifyOutput": false,
+        "disablePlayground": true,
+        "isLearnByExample": false
       }
```

Comment:
Let's place this next to the `Input Objects` BBE.

Reply:
Fixed in [73bf589](https://github.com/ballerina-platform/ballerina-distribution/pull/5188/commits/73bf589184b148f87e83d105386f7ec2710527b7)